hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestSlowPeerTracker.java

```diff
@@ -79,9 +79,12 @@ public void testEmptyReports() {
 
   @Test
   public void testReportsAreRetrieved() {
-    tracker.addReport("node2", "node1", 1.2);
-    tracker.addReport("node3", "node1", 2.1);
-    tracker.addReport("node3", "node2", 1.22);
+    OutlierMetrics outlierMetrics1 = new OutlierMetrics(0.0, 0.0, 0.0, 1.2);
```

Comment:
Can we not define a variable `outlierMetrics1` here, just to make it a little bit cleaner.

Reply:
Sorry I didn't get it. It's already `outlierMetrics1` :)

Reply:
I mean just write it like this:
tracker.addReport("node2", "node1", new OutlierMetrics(0.0, 0.0, 0.0, 1.2));

Reply:
Got it, addressed this change in the latest commit, thanks @tomscut